frontend/src/views/auth/NavigationAuth.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-app-bar
+    v-if="$vuetify.breakpoint.mdAndUp"
+    app
+    clipped-left
+    color="blue-grey darken-4"
+    dark
+  >
+    <logo text />
+    <v-spacer />
+    <v-toolbar-items>
+      <v-btn :href="newsLink" target="_blank" text>
+        <v-icon left>mdi-script-text-outline</v-icon>
+        {{ $tc('global.navigation.news') }}
+      </v-btn>
+    </v-toolbar-items>
+    <v-btn
+      :href="helpLink"
+      target="_blank"
+      fab
+      x-small
+      light
+      absolute
+      top
+      right
+      class="help-button--desktop"
+    >
+      <v-icon>mdi-help</v-icon>
+      <span class="d-sr-only">{{ $tc('global.navigation.help') }}</span>
+    </v-btn>
+  </v-app-bar>
+  <v-btn
```

Comment:
you need to hide the button if the link is not configured (if we want to enable a not configured value)

Reply:
Do you think this is necessary?

Reply:
The termsOfServiceLinkTemplate has the same problem, and @manuelmeister made it consistent.
I opened another issue to fix this: https://github.com/ecamp/ecamp3/issues/4777